data/locale.yml

```diff
@@ -197,6 +197,9 @@ pages:
 _404:
   page_not_found: Page not found.
 
+donors:
+  donors: Donors
```

Comment:
There's a section in locale.yml for page metadata like title, description etc. I think it's called `pages`. Put this there and you won't need to set `req.context.page.title` in the route handler.